src/status_im/ui/screens/accounts/login/views.cljs

```diff
@@ -68,7 +68,7 @@
          {:label             (i18n/label :t/password)
           :placeholder       (i18n/label :t/password)
           :ref               #(reset! password-text-input %)
-          :auto-focus        can-navigate-back? ;;this needed because keyboard overlays testfairy alert
+          :auto-focus        can-navigate-back?
```

Comment:
if there is no test-fairy anymore here should be always `true` and we don't need `can-navigate-back? [:can-navigate-back?]` anymore

Reply:
@flexsurfer fixed